### PR TITLE
Prepare release of 1.6.0

### PR DESCRIPTION
**Pull request by rly**
Prepare the release of 1.6.0. Updated version numbers and some infrastructure.

### CODE REVIEW

File: common/resources.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ groups:
     dtype:
     - name: file_object_id
```
Comment (rly) [inline common/resources.yaml:24]: @mavaylon1 I just noticed that in the "objects" table, we have a field called "object_id". Can we rename this field "file_object_id" to just "object_id" as well?

**Comment by rly** [comment]
@mavaylon1 I made a few more minor changes to the doc strings in `resources.yaml`. Please review them and let me know what you think.

File: common/resources.yaml (hunk 1 above)
Comment (mavaylon1) [reply common/resources.yaml:24]: I like it being explicit, especially since on to_dataframe will have both of those columns.

**Review by mavaylon1 — APPROVED** [review]

File: common/resources.yaml (hunk 1 above)
Comment (rly) [reply common/resources.yaml:24]: Sounds good.

**Pull request merged**